test/instrumentation/logcheck/README.md

```diff
@@ -0,0 +1,4 @@
+This directory contains tool for checking use of unstructured logs in a package. It is created to prevent regression after packages have been migrated to use structured logs.
+
+To run the tool use `go run main.go analyzer.go <package_name>`   
+`e.g go run main.go analyzer.go $KUBE_ROOT/pkg/kubelet/lifecycle/`
```

Comment:
At some point we will need to wire this up in `./hack/verify-logger.sh` or something like that but that can be in a follow-up PR.

Reply:
will do that in followup PR